.github/workflows/coder.yaml

```diff
@@ -208,7 +208,7 @@ jobs:
         run: gotestsum --junitfile="gotests.xml" --packages="./..." --
           -covermode=atomic -coverprofile="gotests.coverage"
           -coverpkg=./...,github.com/coder/coder/codersdk
-          -timeout=3m -count=$GOCOUNT -short -failfast
+          -timeout=5m -count=$GOCOUNT -short -failfast
```

Comment:
non-blocking: I wonder if we should instead just use the `Makefile` target so we don't need to keep this in sync?

Reply:
I think this is a comment on the merge commit since I didn't touch this in my PR. Feel free to fix this in a separate PR